src/ensembl/production/metadata/grpc/config.py

```diff
@@ -18,4 +53,5 @@ class MetadataConfig:
     pool_size = os.environ.get("POOL_SIZE", 20)
     max_overflow = os.environ.get("MAX_OVERFLOW", 0)
     pool_recycle = os.environ.get("POOL_RECYCLE", 50)
-    allow_unreleased = os.environ.get("ALLOW_UNRELEASED", False)
+    allow_unreleased = parse_boolean_var(os.environ.get("ALLOW_UNRELEASED", False))
+    debug_mode = parse_boolean_var(os.environ.get("DEBUG", False))
```

Comment:
Would add a log_level defaulting to `WARNING` the can be updated in config env vars?

Reply:
Done in `service.py`
```
log_level = logging.DEBUG if cfg.debug_mode else logging.WARNING
```

Reply:
Yes, and no. I believe the log level should be configurable (Like INFO, TRACE, etc...) But that's ok like this. :-)

Reply:
I'll improve it in another PR :)